pkg/stores/partition/store.go

```diff
@@ -184,11 +187,14 @@ func (s *Store) List(apiOp *types.APIRequest, schema *types.APISchema) (types.AP
 		}
 	}
 	if list == nil { // did not look in cache or was not found in cache
-		stream, err := lister.List(apiOp.Context(), opts.ChunkSize, opts.Resume)
+		stream, err := lister.List(apiOp.Context(), opts.ChunkSize, opts.Resume, opts.Revision)
 		if err != nil {
 			return result, err
 		}
 		list = listprocessor.FilterList(stream, opts.Filters)
+		if lister.Err() != nil {
```

Comment:
Maybe a comment here stating that we are checking for any errors returned during the listing operation. And that FilterList guarantees the stream has been consumed. 
I just have the fear of things getting moved around since there is no clear dependency on `FilterList(...)`

Reply:
yeah good point, it wasn't even obvious to me at first why I couldn't check the error first